libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -522,11 +523,11 @@ void AsmAnalyzer::warnOnInstructions(solidity::Instruction _instr, SourceLocatio
 			"the Metropolis hard fork. Before that it acts as an invalid instruction."
 		);
 
-	if (_instr == solidity::Instruction::JUMP || _instr == solidity::Instruction::JUMPI)
+	if (_instr == solidity::Instruction::JUMP || _instr == solidity::Instruction::JUMPI || _instr == solidity::Instruction::JUMPDEST)
```

Comment:
I think a jumpdest opcode is different from a label.

Reply:
Why? A jumpdest opcode cannot be placed, only a label can.